src/modules/sdbus/progress.c

```diff
@@ -0,0 +1,171 @@
+/************************************************************\
+ * Copyright 2023 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+/* progress.c - integrate sd_bus with the flux reactor
```

Comment:
Or sdbus_router?

Reply:
I used "progress" because that code is letting the sdbus API's event loop have some time to make progress whenever events/timeouts are pending.  "router" or "bridge" is what the whole flux sdbus module is doing so I was looking for something that communicated event loop integration.  Maybe "reactor" or "eventloop" would be a better choice?

Reply:
yeah, perhaps just eventloop.